up42/__init__.py

```diff
@@ -21,83 +21,64 @@
 """
 
 
-from up42.asset import Asset
-from up42.auth import Auth
-from up42.catalog import Catalog
-from up42.initialization import (
-    initialize_asset,
-    initialize_catalog,
-    initialize_job,
-    initialize_jobcollection,
-    initialize_jobtask,
-    initialize_order,
-    initialize_project,
-    initialize_storage,
-    initialize_tasking,
-    initialize_webhook,
-    initialize_workflow,
+from up42 import asset
+from up42 import auth as up42_auth
+from up42 import (
+    catalog,
+    initialization,
+    job,
+    jobcollection,
+    jobtask,
+    main,
+    order,
+    project,
+    storage,
+    tasking,
+    tools,
+    utils,
+    viztools,
+    webhooks,
+    workflow,
 )
-from up42.job import Job
```

Comment:
These changes break python reexport mechanism. Let's roll them back and disable corresponding pylint rule for the whole file